### ISSUE TITLE
system deadlock

### ISSUE THREAD

**Issue by cnHuaShao** [sajari/docconv#112]
Line 102 103 in this doc. go causes a system deadlock, mainly because the coroutine implemented above failed to add valid data to the channel
```
	body := <-bc
	meta := <-mc
```

err:
```
ConvertDoc: could not read doc: mscfb: bad signature; 43016997712
wvText: exit status 255
```

**Comment by pixge** [comment]
I had the same problem, solved with this: #134

**Comment by jonathaningram** [comment]
Fixed in https://github.com/sajari/docconv/pull/134
